Example/BitmovinConvivaAnalytics/ViewController.swift

```diff
@@ -82,26 +82,81 @@ class ViewController: UIViewController {
 
     var playerConfiguration: PlayerConfiguration {
         let playerConfiguration = PlayerConfiguration()
-        playerConfiguration.sourceItem = sourceItem
+        playerConfiguration.sourceItem = vodSourceItem
+//        playerConfiguration.sourceItem = vodSourceItemStartOffset
+//        playerConfiguration.sourceItem = liveSourceItem
+//        playerConfiguration.sourceItem = liveSourceItemStartOffset
         if adsSwitch.isOn {
             playerConfiguration.advertisingConfiguration = adConfig
         }
+
+        let playbackConfiguration = PlaybackConfiguration()
+        playbackConfiguration.isAutoplayEnabled = true
+        playbackConfiguration.isMuted = true
+        playerConfiguration.playbackConfiguration = playbackConfiguration
         return playerConfiguration
     }
 
-    var sourceItem: SourceItem {
+    var vodSourceItem: SourceItem {
         var sourceString = "https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/m3u8s/11331.m3u8"
         if let streamString = streamUrlTextField.text,
            URL(string: streamString) != nil {
             sourceString = streamString
         }
 
         let sourceItem = SourceItem(url: URL(string: sourceString)!)!
-        sourceItem.posterSource = URL(string: "https://bitmovin-a.akamaihd.net/content/poster/hd/RedBull.jpg")
         sourceItem.itemTitle = "Art of Motion"
         return sourceItem
     }
 
+    var vodSourceItemStartOffset: SourceItem {
+        var sourceString = "https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/m3u8s/11331.m3u8"
+        if let streamString = streamUrlTextField.text,
+           URL(string: streamString) != nil {
+            sourceString = streamString
+        }
+
+        let sourceItem = SourceItem(url: URL(string: sourceString)!)!
+        sourceItem.itemTitle = "Art of Motion"
+        // set start offset
+        let  options: SourceOptions = SourceOptions()
+        options.startOffset = 30
+        options.startOffsetTimelineReference = .start
+        sourceItem.options = options
+        return sourceItem
+    }
+
+    var liveSourceItem: SourceItem {
```

Comment:
Please remove the live source items. The included HLS live URL need not be publicly available. I can also do this before merge.

Reply:
Removed this